tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "target": "es2016",
     "module": "commonjs",
     "outDir": "dist",
+    "noEmitOnError": true,
```

Comment:
why?

Reply:
This is intentional - to prevent incorrect JS emission on type checking failure.